runtime/moonbase/tests/integration_test.rs

```diff
@@ -155,6 +156,17 @@ fn verify_pallet_prefixes() {
 			}
 		]
 	);
+	// Ready to go once we have https://github.com/paritytech/substrate/pull/9246
+	// assert_eq!(
+	// 	<moonbeam_runtime::MaintenanceMode as StorageInfoTrait>::storage_info(),
+	// 	vec![
+	// 		StorageInfo {
+	// 			prefix: prefix(b"MaintenanceMode", b"MaintenanceMode"),
+	// 			max_values: None,
+	// 			max_size: Some(845),
+	// 		},
+	// 	]
+	// );
```

Comment:
we definitely do already have it in our substrate version, but for some reason it never worked for me, I've tried it a few times actually